### ISSUE TITLE
tmp_install.wim, unable to load DLL 'unattend.dll'

### ISSUE THREAD

**Issue by Alexusson** [DrEmpiricism/Optimize-Offline#117]
HI Doc, 
Can you direct me on how to solve this beginner problem? I am getting this error message
Thanks in advance

ErrorRecord.log 
**Dealing with the .ISO file**
---- -----                                                                                                             
 462 Adding 'tmp_install.wim' would result in a result image having a size larger than the current configured limit.   
                                                                                                                       
1994 The term 'Stop-Optimze' is not recognized as the name of a cmdlet, function, script file, or operable program.    
     Check the spelling of the name, or if a path was included, verify that the path is correct and try again.         
1985 Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)  
1928 The term 'Stop-Optimze' is not recognized as the name of a cmdlet, function, script file, or operable program.    
     Check the spelling of the name, or if a path was included, verify that the path is correct and try again.         
1922 Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)  
1890 The term 'Stop-Optimze' is not recognized as the name of a cmdlet, function, script file, or operable program.    
     Check the spelling of the name, or if a path was included, verify that the path is correct and try again.         
1882 Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)

**Dealing with the .WIM file**
---- -----                                                                                                             
1994 The term 'Stop-Optimze' is not recognized as the name of a cmdlet, function, script file, or operable program.    
     Check the spelling of the name, or if a path was included, verify that the path is correct and try again.         
1985 Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)  
1928 The term 'Stop-Optimze' is not recognized as the name of a cmdlet, function, script file, or operable program.    
     Check the spelling of the name, or if a path was included, verify that the path is correct and try again.         
1922 Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)

**Comment by Alexusson** [comment]
Actuially I use all same configuration as before: https://github.com/DrEmpiricism/Optimize-Offline/issues/108

**Copied from the Start.cmd**
for WIM files
SET "SourcePath=C:\Optimize-Offline-master\MOUNT\sources\install.wim"
OR for ISO files
SET "SourcePath=C:\Optimize-Offline-master\ISO\en_windows_10_enterprise_2016_ltsb_x64_dvd_9059483.iso"

    PowerShell.exe -NoProfile -ExecutionPolicy Bypass -File .\Optimize-Offline.ps1 -SourcePath "%SourcePath%" -WindowsApps "Whitelist" -SystemApps -Packages -Features -Win32Calc -Registry -Additional -ISO "Prompt"

**Comment by DrEmpiricism** [comment]
Have you tried the latest build yet to see if this error still occurs?

**Comment by Alexusson** [comment]
Hi, doc,

Sorry for the delay. Unfortunately, I am getting the same result. Tried on both version 3.2.6.9 and 3.2.7.0.

Below are last relevant rows from the Optimize-Offline.log file:
2019-09-06T11:42:25 [ERROR]: Failed to Export Microsoft Windows Setup (x64)
2019-09-06T11:42:25 [ERROR]: Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E) (System.DllNotFoundException,Microsoft.Dism.Commands.ExportWindowsImageCommand: C:\Optimize-Offline-master\Optimize-Offline.ps1:1939 char:20)
2019-09-06T11:42:28 [INFO]: Rebuilding and Exporting Microsoft Windows Recovery Environment (x64)
2019-09-06T11:42:44 [ERROR]: Failed to Export Microsoft Windows Recovery Environment (x64)
2019-09-06T11:42:44 [ERROR]: Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E) (System.DllNotFoundException,Microsoft.Dism.Commands.ExportWindowsImageCommand: C:\Optimize-Offline-master\Optimize-Offline.ps1:1967 char:16)
2019-09-06T11:42:53 [INFO]: Saving and Dismounting Windows 10 Pro
2019-09-06T11:50:06 [INFO]: Rebuilding and Exporting Windows 10 Pro using Maximum compression.
2019-09-06T11:54:35 [ERROR]: Failed to Export Windows 10 Pro
2019-09-06T11:54:35 [ERROR]: Unable to load DLL 'unattend.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E) (System.DllNotFoundException,Microsoft.Dism.Commands.ExportWindowsImageCommand: C:\Optimize-Offline-master\Optimize-Offline.ps1:2030 char:16)
2019-09-06T11:54:35 [INFO]: Dismounting and discarding the image.

**Comment by DrEmpiricism** [comment]
This is a message you get when you run PowerShell x86 on 64-bit Windows.

**Comment by Alexusson** [comment]
Hi Doc,

Finally, I caught some time. I will share my experience here, and maybe it will be useful and give you some ideas.

Meanwhile, from my last reply and problem above, I had one more problem with running Start.cmd as an administrator. It refused to run as administrator... So I decided to do a clean install of another OS on my machine. The script now works fine, the same as before, and the modified PRO build is perfectly stable! I still cannot get functional LTSC which is frustrating because I saw people made stable LTSC even after they remove much more packages than me!

Almost forgot, even if I Enabled EFI (special OSes only) as I did here https://github.com/DrEmpiricism/Optimize-Offline/issues/108 modified builds cannot be installed in VirtualBox anymore (version 3.2.7.2). So now I am using VMWare Workstation, and there is no issue.
Are modified builds Bios or UEFI bootable or both?? I guess UEFI.

I see you removed the .ps1 script for removing OneDrive from Setup folder. I think it is the right solution because I had a problem with the script. Command SFC /scannow stops at 45-47% and shows some error each time after I use this script to remove OneDrive.

Another "issue" related to OneDrive is persisting and can be seen on the first login (the latest version 3.2.7.2). OneDrive is still present in Task Manager with incredible high CPU usage (almost 100%), but after rebooting the system, he is gone forever ;)

I like your flexible and time-effective tool a lot, and looking forward to testing your further releases. I read somewhere that we can expect more sweets from you, versions with GUI and additional features.

Till then. More power to your brain and skills.

Cheers,
Alex
